images/build-libs/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM debian:stable
```

Comment:
maybe we could pin a version?

Reply:
I haven't pinned a version because there's not yet a clear requirement on falcosecurity/libs about that. I believe once we clean up and fine-tune the build system, it will become evident what image should be used and consequently an image version to be pinned.
Basically, just assume `debian:stable` as temporary solution :)

Reply:
Oh ok sure :) Thanks for the explanation